INFO:     Uvicorn running on http://0.0.0.0:5000 (Press CTRL+C to quit)
INFO:     Started reloader process [78940]
2020-07-17 19:22:49.648756: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
Process SpawnProcess-1:
Traceback (most recent call last):
  File "C:\Users\mahjo\AppData\Local\Programs\Python\Python36\Lib\multiprocessing\process.py", line 258, in _bootstrap
    self.run()
  File "C:\Users\mahjo\AppData\Local\Programs\Python\Python36\Lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\uvicorn\subprocess.py", line 61, in subprocess_started
    target(sockets=sockets)
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\uvicorn\main.py", line 382, in run
    loop.run_until_complete(self.serve(sockets=sockets))
  File "C:\Users\mahjo\AppData\Local\Programs\Python\Python36\Lib\asyncio\base_events.py", line 484, in run_until_complete
    return future.result()
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\uvicorn\main.py", line 389, in serve
    config.load()
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\uvicorn\config.py", line 288, in load
    self.loaded_app = import_from_string(self.app)
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\uvicorn\importer.py", line 20, in import_from_string
    module = importlib.import_module(module_str)
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File ".\text_classification\app.py", line 26, in <module>
    metric="test_loss", objective="minimize")
  File ".\text_classification\utils.py", line 109, in get_best_run
    for run in runs:
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\wandb\apis\public.py", line 583, in __next__
    if not self._load_page():
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\wandb\apis\public.py", line 571, in _load_page
    self.objects.extend(self.convert_objects())
  File "c:\users\mahjo\e2e-ml-app-tensorflow\venv\lib\site-packages\wandb\apis\public.py", line 736, in convert_objects
    for run_response in self.last_response['project']['runs']['edges']:
TypeError: 'NoneType' object is not subscriptable
